manifests/patch-e2e.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
```

Comment:
Is this script used for k8s less than `v1.13`?

Reply:
> Is this script used for k8s less than `v1.13`?

Yes, in e2e test with 1.12 k8s version, we had to declare cabundle.

Reply:
to make this clear, I think we should run this script only if the k8s version is less than 1.13. here is an example on how to compare k8s version (https://github.com/pingcap/tidb-operator/blob/eb11bce9107ff7f0f484e8fb8cfe5d6b8cb10daf/tests/e2e/tidbcluster/tidbcluster.go#L128-L132)

Reply:
Thanks! I would enhance it next issue.

Reply:
create an issue or a TODO item for this

Reply:
issued in https://github.com/pingcap/tidb-operator/issues/1317